### ISSUE TITLE
More options for when you can drag

### ISSUE THREAD

**Issue by Pederytter** [donmbelembe/vue-dragscroll#4]
Hey again!

So I finally came around to this problem again. I'm not sure if you have a work around for it, but my issue is that I wan't multiple html elements to be dragable to the parent element, which have v-dragable:nochilddrag.

I know v-dragable makes all child elements dragable, but this isn't what I'm looking for either. I might have 10 child elements from the element which has v-dragable, where only 5 of them needs to be dragable from the parent element.

A simple (not so great, but hope you get the idea from it) can be seen here.

https://codepen.io/anon/pen/RjbJdJ

What I'm trying is to make the whole green area dragable, without making the blue areas with input dragable. I know I can place my boxes out of the columns, but that can't be done in what I'm trying to accomplish in the project I'm working on.

Any idea how this can be done or if it's even possible with the current version?

**Comment by donmbelembe** [comment]
You want the all green area dragable even between blue boxes ?

**Comment by donmbelembe** [comment]
if yes, the problem your are facing come from the `area-inside-box` element because you set `nochilddrag`

**Comment by Pederytter** [comment]
Exactly, I wan't the green area to be dragable, but not the blue area. That's why I need to use nochilddrag, but that doesn't let me use the all the green area.

So my question was, if there was a way this could be accomplish without changing the html structure?

**Comment by donmbelembe** [comment]
let me publish for u an update with a new feature that will solve your problem

**Comment by donmbelembe** [comment]
i have published the version 1.3.1
you can use` :firstchilddrag`  https://codepen.io/anon/pen/ZaEpqq (check the docs)

here is your structure

```
app
-- scroll-box (v-dragscroll)
-- -- area-inside-box
-- -- -- column
-- -- -- -- box
-- -- -- -- box
-- -- -- column
-- -- -- -- box
-- -- -- -- box
```

using `:nochilddarg`, only the `scroll-box` will be dragable, the `area-inside-box` and everything inside will not dragscroll.
in the latest version, I've published the `:firstchilddarg` this will make the element (`scroll-box`) and it first child (`area-inside-box`) dragable but not the remaining grand childs 

but with your structure you also want to support the `column` element.
I will think about an update for that later.

but you can do something,
use v-dragscroll without any argument (this will make disable your inputs) then add an event listener to focus your inputs (on click)

**Comment by Pederytter** [comment]
Thanks a lot!
I found a way to patch my project with this new implementation.
